data/vector.go

```diff
@@ -159,8 +159,11 @@ func ValidFieldType(t interface{}) bool {
 type FieldType int
 
 const (
+	// FieldTypeUnknown indicates that we do not know the field type
+	FieldTypeUnknown FieldType = iota
+
 	// FieldTypeInt8 indicates the underlying primitive is a []int8.
-	FieldTypeInt8 FieldType = iota
```

Comment:
In general it's possible to use `iota + 1` to start counting from one - but I think `FieldTypeUnknown` has more sense here 👍